keyboards/scottokeebs/scotto34/info.json

```diff
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "Scotto34 (PCB Edition)",
+    "maintainer": "joe-scotto",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": [
+            "GP9",
+            "GP10",
+            "GP11",
+            "GP12",
+            "GP13",
+            "GP18",
+            "GP19",
+            "GP20",
+            "GP21",
+            "GP22"
+        ],
+        "rows": ["GP0", "GP1", "GP14", "GP15"]
+    },
+    "processor": "RP2040",
+    "url": "https://scottokeebs.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x3334",
+        "vid": "0x534B"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
Utterly optional, but changing `LAYOUT` to `LAYOUT_split_3x5_2` will allow community layouts to automatically be assigned (namely miryoku).
Would have to apply this change to all keymap.c files also.

Reply:
I like that change, I added it to the board and will be for all the other ones you've been tagging :)